adapters/repos/db/shard_test.go

```diff
@@ -163,3 +165,52 @@ func TestShard_ReadOnly_HaltCompaction(t *testing.T) {
 	require.Nil(t, idx.drop())
 	require.Nil(t, os.RemoveAll(idx.Config.RootPath))
 }
+
+var parallelTests = []struct {
+	name            string
+	goroutineFactor []string
+}{
+	{"Valid factor", []string{"1"}},
```

Comment:
would be good to add in more test cases to ensure other valid factors work aside from `1`

Reply:
added two more